tests/forms_tests/tests/test_forms.py

```diff
@@ -501,6 +501,22 @@ class SongForm(Form):
 <option value="P" selected="selected">Paul McCartney</option>
 </select>""")
 
+    def test_form_with_disabled_fields(self):
+        class PersonForm(Form):
+            name = CharField()
+            birthday = DateField(disabled=True, initial=datetime.date(1974, 8, 16))
+
+        # Disabled fields are generally not transmitted by user agents
+        f = PersonForm({'name': 'John Doe'})
+        self.assertTrue(f.is_valid())
+        self.assertEqual(f.cleaned_data, {'birthday': datetime.date(1974, 8, 16), 'name': 'John Doe'})
+
+        # Even if someone is tricking post data to pass a different value, we
```

Comment:
`# Values provided in the form's data are ignored.`

Might be good to have a test for `Form(data, initial=...)` too.